packages/components/pages/f-checkout/test/accessibility/axe-accessibility.spec.js

```diff
@@ -170,11 +171,11 @@ describe('Accessibility tests', () => {
             lastName: 'Jazzman',
             emailAddress: 'jerry.jazzman@ronniescotts.co.uk',
             mobileNumber: '+447111111111',
-            line1: '1 Bristol Road',
-            locality: 'Bristol',
-            postcode: 'BS1 1AA'
+            addressLine1: '1 Bristol Road',
+            addressLocality: 'Bristol',
+            addressPostcode: 'BS1 1AA'
         };
-        checkout.populateCheckoutForm(checkoutInfo, customerInfo);
+        Object.keys(customerInfo).forEach(field => checkout.setFieldValue(field, customerInfo[field]));
```

Comment:
Nit: please add an empty line before and after this line (and the same applies to the other files 🙂)

Reply:
Besides that, given you have this everywhere, it may be worth extracting it to its own function. Not sure. I'll leave it up to you.